lib/luaxtable.h

```diff
@@ -0,0 +1,16 @@
+/*
+* SPDX-FileCopyrightText: (c) 2024 Mohammad Shehar Yaar Tausif <sheharyaar48@gmail.com>
+* SPDX-License-Identifier: MIT OR GPL-2.0-only
+*/
+
+#ifndef luaxtable_h
+#define luaxtable_h
+
+struct luaxtable_s;
+
+typedef struct luaxtable_info_s {
+	struct luaxtable_s *data __attribute__((aligned(8)));
```

Comment:
can we use this to pass "generic" parameters from the userspace? similarly to what we are doing on eBPF -> luaxdp? I would add an parm in your example.. perhap the domain to be blocked.. or a the name of a lua table containing the blacklist.. (or anything else ;-)

Reply:
We can pass parameters to this module, that would be covered in the user-library generation.